docs/developer-guide/services.md

```diff
@@ -223,10 +223,10 @@ Each Prowler service requires a service client to use the service in the checks.
 The following is the `<new_service_name>_client.py` containing the initialization of the service's class we have just created so the service's checks can use them:
 
 ```python
-from prowler.providers.<provider>.lib.audit_info.audit_info import audit_info
+from prowler.providers.common.common.get_global_provider import set_mocked_<provider>_provider
 from prowler.providers.<provider>.services.<new_service_name>.<new_service_name>_service import <Service>
 
-<new_service_name>_client = <Service>(audit_info)
+<new_service_name>_client = <Service>(set_mocked_<provider>_provider([<region>]))
```

Comment:
Since the function to get a mocked provider has a lot of arguments I'll point that since the `region` is not the only used.

Reply:
Changed it to the right structure so it's not necessary at all